x/claim/keeper/claim.go

```diff
@@ -152,7 +150,7 @@ func (k Keeper) GetAirdropByIdentifier(ctx sdk.Context, airdropIdentifier string
 func (k Keeper) GetDistributorAccountBalance(ctx sdk.Context, airdropIdentifier string) (sdk.Coin, error) {
 	airdrop := k.GetAirdropByIdentifier(ctx, airdropIdentifier)
 	if airdrop == nil {
-		return sdk.Coin{}, errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, "invalid airdrop identifier: GetDistributorAccountBalance")
+		return sdk.Coin{}, sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid airdrop identifier: GetDistributorAccountBalance")
```

Comment:
Should we use `errorsmod` or `sdkerrors` in general? I see [other files](https://github.com/Stride-Labs/stride/blob/main/x/stakeibc/keeper/icacallbacks_reinvest.go) use `errorsmod`, so just want to make sure this is intentional!

Reply:
We should use errorsmod anytime we’re calling Wrapf or Register

sdkerrors should be used when referring to an error type (such as sdkerrors.ErrInvalidAddress)

Reply:
Updated - what's the rationale for using one vs. the other out of curiosity?

Reply:
sdkerrors.Wrapf and sdkerrors.Register are deprecated! So they’ll cause lint errors which will block the PR from being merged l.

similar to sdk.Int → sdkmath.Int